lib/formtools/plugins/plugins-helpers.js

```diff
@@ -226,6 +226,41 @@ var overviewDefaults = function (overview, labels) {
 
     overview.summary.label = overview.summary.label || 'Summary';
 
+
+    //overview Details
+    if (!overview.details || typeof overview.details !== 'object') {
+        // defaults to {}
```

Comment:
done. Code to process overview.details has been removed from plugins-helpers.js. It has been moved to formtools.overview.